libraries/botbuilder-dialogs-adaptive-teams/package.json

```diff
@@ -31,7 +31,11 @@
     "adaptive-expressions": "4.1.6",
     "botbuilder": "4.1.6",
     "botbuilder-dialogs": "4.1.6",
-    "botbuilder-dialogs-adaptive": "4.1.6"
+    "botbuilder-dialogs-adaptive": "4.1.6",
+    "botbuilder-dialogs-declarative": "4.1.6",
+    "botbuilder-stdlib": "4.1.6",
+    "botframework-schema": "4.1.6",
```

Comment:
We shouldn't have a `botframework-schema` dependency, `botbuilder` re-exports `botframework-schema`'s exports